### PR TITLE
Scrub dat mobile data

### LINKED ISSUE #6409: Fix issue where multiple Drupal users share the same email.
#### Current Behavior

There are currently a bunch of users (both old and new) that are registered with the same value in the `mail` column in the users table. This is kinda a bummer since we should really only have one account per email. 😞 📬 
#### Desired Behavior

We need to do some more research on how Drupal is currently treating users with duplicate emails (like when you try to log in with that email, which account does it pick?), and then remove the duplicate users who are not being actively used (and perhaps merge their activity into the "active" accounts?)
#### Steps to Replicate

Try running this real outrageous 🐬 query:

``` sql
SELECT group_concat(u.uid), u.name, u.mail, m.field_mobile_value, s.field_user_registration_source_value, from_unixtime(u.access), count(u.mail)
FROM users u
LEFT JOIN field_data_field_mobile m on m.entity_id = u.uid 
left join field_data_field_user_registration_source s on s.entity_id = u.uid
GROUP BY u.mail
HAVING COUNT(u.mail) > 1
ORDER BY u.mail
```
#### Why This Matters

When trying to sync users with Northstar, we're getting validation errors because we're trying to make multiple users that all share the same "unique" field. It's also super confusing how this even _works_ in Phoenix so it'd be nice to figure out that mystery!

📪 🌵

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?

Adds a script to cleans up mobile data for users ported from the old days database.
#### How should this be reviewed?

📜 
#### Any background context you want to provide?

nah. 
#### Relevant tickets

Fixes #6409
#### Checklist
- [ ] Tested on staging.

### CODE REVIEW

File: scripts/sanitize-mobile-numbers.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                        FROM field_data_field_mobile
+                        WHERE field_mobile_value NOT REGEXP '^[0-9]+$';");
+
+
+foreach($wild_typers as $wilder) {
+  if ($wilder->mobile) {
+    $fresh_and_clean_digits = dosomething_user_clean_mobile_number($wilder->mobile);
+    if ($fresh_and_clean_digits) {
+      $user = user_load($wilder->uid);
+      $edit = ['field_mobile' => [ LANGUAGE_NONE => [ 0 => [ 'value' => $fresh_and_clean_digits ] ] ] ];
```
Comment (DFurnes) [inline scripts/sanitize-mobile-numbers.php:21]: `[ => [ => [ => [ =>`

this is _classic_ droops

**Comment by DFurnes** [comment]
🚢

File: scripts/sanitize-mobile-numbers.php (hunk 1 above)
Comment (angaither) [inline scripts/sanitize-mobile-numbers.php:21]: 😂

Comment (weerd) [inline scripts/sanitize-mobile-numbers.php:21]: sweep it under the rug!

**Pull request merged**